conf/deployment/aws/rosa_hcp_1az_3w_m5.12x.yaml

```diff
@@ -1,13 +1,26 @@
 DEPLOYMENT:
   allow_lower_instance_requirements: false
+  sts_enabled: true
+  force_download_ocm_cli: true
+  force_download_rosa_cli: true
+  ocm_cli_version: '0.1.67'
+  rosa_cli_version: '1.2.45'
+  live_deployment: true
+  customized_deployment_storage_class: 'gp3-csi'
 RUN:
-  username: 'cluster-admin'
+  username: "cluster-admin"
 ENV_DATA:
   platform: 'rosa_hcp'
-  deployment_type: 'managed'
```

Comment:
I think you should preserve the `deployment_type` parameter in the config too, because otherwise it is [set by default to `ipi`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L155), which is not correct in this case, so I think it will be better to have it explicitly configured. (It is quite heavily used parameter.)

Reply:
Thank you for the review, and important question. 

The problem is that if deployment_type is set to 'managed', it will fail in a large number of 'if' blocks, both within the deployment and tests. The platform behaves as both managed and self-managed. From my perspective, it was presented as self-managed because the user will have access to metrics, alerts, and the ability to manage storage independently. Additionally to this, user can manage HCP nodes independently. So the support is for control-plane machines/nodes and k8s resources.

Epic: [ODF self-managed installable on ROSA HostedControlPlane (HCP)](https://issues.redhat.com/browse/RHSTOR-4858)
The experience should be as close to regular ODF on AWS as possible.

This left me with open question, should we present another deploment_type, such as "semi_managed"?

Reply:
Thanks for the explanation. It will be probably better to introduce new type, because otherwise it will be considered as `ipi` (due to the default value for this parameter), which is not correct and might be misleading.